Classes/Service/ContentService.php

```diff
@@ -389,7 +389,17 @@ public function fixPositionInLocalization($uid, $languageUid, &$sourceRecord, Da
                 $previousLocalizedRecordUid
             );
         }
-        $this->updateRecordInDataMap($localizedRecord[0], null, $reference);
+
+        $localizedRecord[0]['colPos'] = $sourceRecord['colPos'];
+
+        $parentLocalizedRecord = BackendUtility::getRecordLocalization('tt_content', $sourceRecord['tx_flux_parent'], $languageUid);
+        $localizedRecord[0]['tx_flux_parent'] = $parentLocalizedRecord[0]['uid'];
+        $localizedRecord[0]['tx_flux_column'] = $sourceRecord['tx_flux_column'];
+
+        $this->recordService->update('tt_content', $localizedRecord[0]);
```

Comment:
This causes an additional SQL update query - please use the `updateRecordInDataMap` method to change the values in DataHandler which then saves them for us later.